projects/docker.com/cli/package.yml

```diff
@@ -0,0 +1,40 @@
+distributable:
+  url: https://github.com/docker/cli/archive/v{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: docker/cli/tags
+platforms:
+  # colima -> only darwin
+  - darwin
```

Comment:
we should build for linux and make the dependency darwin only?

and well, the dep isn't a dep really. It's a `companion` at best.

companions cannot be platform restricted tho.

will do for now to have it as a dep. but yeah we should build for linux if poss.